Stage View for Images in Adaptive cards (#2952)
MicrosoftDocs/msteams-docs: msteams-platform/task-modules-and-cards/cards/cards-format.md

If you have not set the `width` property to *Full*, then the default view of the Adaptive Card is as follows:
![Small width Adaptive Card view](../../assets/images/cards/small-width-adaptive-card.png)

> ### Stage View for images in Adaptive cards

Reviewer: Please change to:

```suggestion
### Stage view for images in Adaptive Cards
```

> You can also use the `msteams` property to selectively add the ability to enlarge images, in an Adaptive card, in stage view. Users would see an expand icon when they hover over the images, for which the `allowExpand` attribute is set to `true`. For information on how to use the property, see the following example:

Reviewer: ```suggestion
In an Adaptive card, in stage view, you can use the `msteams` property to add the ability to enlarge images selectively. When users hover over the images, they would see an expand icon, for which the `allowExpand` attribute is set to `true`. For information on how to use the property, see the following example:
```

Author: this property is added  which enables users to see the image in stage view. Updated to reflect the same. please review and let me know if this doesnt look right.

``` json
{
    "type": "AdaptiveCard",
     "body": [
          {
            "type": "Image",
            "url": "https://picsum.photos/200/200?image=110",
            "msTeams": {
              "allowExpand": true
            }
          },
     ],
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
    "version": "1.2"
}
```

When users hover over the image, they would see an expand icon at top right corner of the image.
![Adaptive card with expandable image](../../assets/images/cards/adaptivecard-hover-expand-icon.png)

> The image would be shown in stage view when clicking on the expand button.

Reviewer: ```suggestion
The image is shown in stage view when the user clicks on the expand button.
```

![Image expanded to stage view](../../assets/images/cards/adaptivecard-expand-image.png)

> In the stage view, users would be able to zoom in and zoom out of the image.
> You can choose which images in your Adaptive card need to have this capability.
> Please note that this ability applies only to image elements (Image type) in an Adaptive card.

Reviewer: ```suggestion
In the stage view, users can zoom in and zoom out of the image. You can select which images in your Adaptive card needs to have this capability.

> [!NOTE]
> Zoom in and zoom out capability applies only to the image elements (Image type) in an Adaptive card.
```
